### PR TITLE
UIORGS-34 Create and Edit interface

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
Purpose: Allow users and other applications to manage interfaces and assign them to organizations.
https://issues.folio.org/browse/UIORGS-34

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Interface/InterfacesList.js [hunk 1]
```diff
@@ -128,7 +128,6 @@ const InterfacesList = ({ fetchInterfaces, fields, interfaces, orgId, stripes })
         formatter={resultsFormatter}
         rowFormatter={anchoredRowFormatter}
         visibleColumns={visibleColumns}
-        rowMetadata={orgId}
```
Comment (aliaksei-chumakou) [inline src/Interface/InterfacesList.js]: I don't think it's used, but it produces a warning in browser console, so I've decided to remove it

File: src/Interface/InterfacesList.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ const AddInterfaceButton = ({ fetchInterfaces, fields, stripes, orgId }) => {
 };
 
 AddInterfaceButton.propTypes = {
-  fetchInterfaces: PropTypes.func.isRequired,
+  fetchInterfaces: PropTypes.func,
```
Comment (NikitaSedyx) [inline src/Interface/InterfacesList.js]: should be default prop added?

File: src/common/styles/subHeadings.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.subHeadings {
+  color: var(--subHeading-color);
```
Comment (NikitaSedyx) [inline src/common/styles/subHeadings.css]: how does it work without import?

File: src/Interface/InterfacesList.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/Interface/InterfacesList.js]: I'll revert this

File: src/common/styles/subHeadings.css (hunk 3 above)
Comment (aliaksei-chumakou) [reply src/common/styles/subHeadings.css]: magic https://github.com/folio-org/ui-organizations/blob/master/src/Main/Main.css

File: test/bigtest/tests/editInterface.e2e.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    expect(page.saveButton.isDisabled).to.be.true;
+  });
+
+  describe('Save interface', () => {
+    beforeEach(async function () {
+      await page.name.fill('new name');
+      await page.saveButton.click();
+    });
+
+    it.only('interface form is closed', () => {
```
Comment (NikitaSedyx) [inline test/bigtest/tests/editInterface.e2e.js]: `only`

**Pull request merged**